xml/article_setup.xml

```diff
@@ -1452,13 +1455,13 @@ rmdir /dev/cpuset/$1
       separate I/O queue for each process. All of these queues get a similar
       time slice for disk access. With this procedure, the
       <emphasis>CFQ</emphasis> tries to divide the bandwidth evenly between all
-      requesting processes. This scheduler has a similar throughput as the
+      requesting processes. This scheduler has a similar throughput to the
```

Comment:
Maybe make sure "similar" + "to" are together and maybe there's a better replacement for "has" that also expresses that throughput can vary depending on requirements?
Suggestion: `This scheduler allows for throughput similar to [...]`